### PR TITLE
202: Payment reminders

### PR DESCRIPTION
**Pull request by Nesssu**
Membersship payment reminders. First reminder will be sent when the memebrship will expire in two weeks, second one when the expiration is in a week and the final reminder will be send on the day it expires. 

This is a bit harder to test because your membership is probably not expiring soon, but if you want to test it you can manually change your membership expiration date from Mongoose.

Also we can't demo this during the presentation so the important thing is that we can say we have it and there's code for it :)

### CODE REVIEW

**Comment by Nesssu** [comment]
[202 - Payment reminders](https://trello.com/c/YoPNEOUv/128-202-payment-reminders)

File: server/routes/api.js [hunk 1]
```diff
@@ -755,4 +741,117 @@ cron.schedule("0 0 * * * *", async () =>
...
+            }
+            else if (member.membershipExpirationDate.getDate() == new Date().getDate())
+            {
+                expiresToday.push(member.email);
+            }
+        })
+
+        if (expiresToday.length > 0) 
+        {
+            const message = "Your membersship to AssocEase expires today!"
```
Comment (jannejjj) [inline server/routes/api.js:782]: A small typo in "membersship", should only have one s :D Applies to all messages

**Review by jannejjj — APPROVED** [review]
Check the typo I commented on, but otherwise looks good! Didn't actually test though.

File: server/routes/api.js (hunk 1 above)
Comment (Nesssu) [reply server/routes/api.js]: Fixed that!

**Review by Jezerig — APPROVED** [review]

**Pull request merged**